docs/src/terminology.md

```diff
@@ -8,6 +8,10 @@ The following terms are used throughout the documentation.
 
 A persistent file addressed by [public key](terminology.md#public-key) and with [lamports](terminology.md#lamport) tracking its lifetime.
 
+## active set
+
+The list of validators that can be considered for the next epoch’s leader schedule
```

Comment:
can be considered?  also, link to `[validators](terminology.md#validator)`, and end with a period

Reply:
Correct me if I'm wrong, but is there anything that ensures that every validator is given a slot for each epoch? If, for example, you enable validating through a Playstation 5 and hundreds of thousands of people enable this, would there not be enough slots? Or if another cluster spun up with very short epics, but had lots of validators, could there not be some validators missing from the leader schedule?

Reply:
Thanks for that context. How about "The set of non-delinquent validators; each is eligible for a slot in the next epoch's leader schedule."?  @carllin, is that accurate?

Reply:
@garious, they can be delinquent, they just have to be staked :)

Currently leader schedule generation (we think of validators in the leader schedule as the "active set"), in `leader_schedule_utils::leader_schedule` does not take into account voting behavior

Reply:
Thanks @carllin. So is the active set the staked validators or the subset that's in the leader schedule?

Reply:
sorry! I should have been more clear.

Looking at these docs, the definition and references to the "active set" is closely related to voting (i.e. you are staked and voting), which is not actually implemented in the code.

If we want to maintain the current definition, where the active set for epoch `X` is the list of candidates for epoch `X+2`'s  leader schedule, then that would just be the staked validators in epoch `X`, which is not necessarily the subset in leader schedule of the epoch `X` as some validators may have unstaked/joined.